front/app/containers/Admin/reporting/containers/ReportBuilder/index.tsx

```diff
@@ -115,7 +122,7 @@ const ReportBuilder = ({ reportId, reportLayout }: Props) => {
     0;
 
   return (
-    <ReportContextProvider width="pdf" reportId={reportId}>
+    <ReportContextProvider width="pdf" reportId={reportId} phaseId={phaseId}>
```

Comment:
I had to rewrite the `ReportContextProvider` a bit. Before, you would pass it the `reportId`, which would be used to fetch the report, and there we would check if there is a phase relation. But because we also use this provider for the rendered report under the phase, and because normal users don't have access to reports, this would stop the whole report from rendering. Now, we instead derive the phase outside of the `ReportContextProvider`.

Reply:
Indeed, the report is not shown to citizens :(
When did it break? How did we miss it? Maybe we should do at least some simple smoke tests before releases?

Reply:
We just need e2e tests I think